ext/app/electron/config.ts

```diff
@@ -0,0 +1,206 @@
+import * as path from 'path'
+import * as fse from 'fs-extra'
+import * as electron from 'electron';
+import * as ini from 'ini'
+import * as os from 'os'
+import * as log from 'app/server/lib/log';
+import * as packageJson from 'desktop.package.json';
+import {commonUrls} from 'app/common/gristUrls';
+
+const CONFIG_DIR = path.join(electron.app.getPath("appData"), packageJson.name)
+
+const APPDATA_DIR = (process.platform == "win32") ? electron.app.getPath("userData"):
+  path.join(electron.app.getPath("home"), ".local", "share", packageJson.name)
+
+// Electron's app.getPath("userData") uses productName instead of name
+// but productName should be "full capitalized name", not ideal for naming our config directory.
+const DEFAULT_CONFIG_FILE = path.join(CONFIG_DIR, "config.ini")
```

Comment:
Hmm there's linting set up for grist-core that isn't set up on this repo yet. We use semicolons. In general, try to go with the style of neighboring files.

Reply:
I'll actually take some time to set up linting.

Reply:
Basic linter is up now.